notebooks/ch12/linear_tips.md

```diff
@@ -59,23 +59,25 @@ Although the average tip amount is \$3, if a table orders \$40 worth of food we
 
 Let's briefly review our current toolbox for modeling and estimation and define some new notation so that we can better represent the additional complexity that linear models have.
 
-## Defining a Linear Model
+## Defining a Simple Linear Model
 
-We are interested in predicting the tip amount based on the total bill of a table. We use $ y $ to represent the tip amount, the variable we are trying to predict. We use $ x $ to represent the total bill, the variable we use for prediction. Our model assumes that in truth, $ y $ has a perfectly linear relationship with $ x $. However, our observed data do not follow a perfectly straight line because of some random noise $ \epsilon $. Mathematically, we write:
+We are interested in predicting the tip amount based on the total bill of a table. We use $ y $ to represent the tip amount, the variable we are trying to predict. We use $ x $ to represent the total bill, the variable we use for prediction. 
+
+We define a linear model $ f_\theta $ that depends on $ x $:
 
 $$
-y = f_\theta (x) + \epsilon
+f_\theta (x) = \theta_1 x + \theta_0
 $$
 
-where
+and treat it as the true generating function of the data.
+
+$ f_\theta (x) $ assumes that in truth, $ y $ has a perfectly linear relationship with $ x $. However, our observed data do not follow a perfectly straight line because of some random noise $ \epsilon $. Mathematically, we account for this by adding a noise term:
```

Comment:
nit: "generating function" has a specific mathematical meaning (https://www.wikiwand.com/en/Generating_function). Also, we should capitalize the first word of sentences after an equation.

Let's instead say "We treat $ f_\theta (x) $ as the underlying function that generated the data".